src/vs/workbench/services/textfile/browser/textFileService.ts

```diff
@@ -370,9 +374,9 @@ export abstract class TextFileService implements ITextFileService {
 		return this._models.getAll(<URI>arg1);
 	}
 
-	private getDirtyFileModels(resources?: URI[]): ITextFileEditorModel[];
-	private getDirtyFileModels(resource?: URI): ITextFileEditorModel[];
-	private getDirtyFileModels(arg1?: any): ITextFileEditorModel[] {
+	public getDirtyFileModels(resources?: URI[]): ITextFileEditorModel[];
```

Comment:
I think that's just left over from experimentation, thanks for spotting it. https://github.com/Microsoft/vscode/pull/14115/commits/e0b7451d463891097c73c5d25d9e8b15f3e03faa